multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ModuleHooksAggregator.java

```diff
@@ -41,8 +38,8 @@ private List<Hook> collectHooksWithPhase(Module moduleToDeploy, List<HookPhase>
                                              .collect(Collectors.toList());
     }
 
-    private List<Hook> getModuleHooks(Module moduleToDeploy) {
-        return moduleToDeploy.getMajorSchemaVersion() < MAJOR_SCHEMA_VERSION_THREE ? Collections.emptyList() : moduleToDeploy.getHooks();
+    private List<Hook> getModuleHooks(Module module) {
+        return SchemaVersionUtils.getEntityData(module, Module::getMajorSchemaVersion, Module::getHooks);
```

Comment:
I don't really like this change - makes it harder to read and doesn't seem to be overridden anywhere

Reply:
Reverted.